test/sagemaker/conftest.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
should this be under `test/integration/sagemaker` to match the other repos?

Reply:
> renaming test/functional to test/sagemaker or renaming everything to test/integration/{local, sagemaker} would fit the convention better

I went with the option that involves fewer changes. but I'm also okay to put everything under test/integration/... 😺

Reply:
if you do move it, do it in a different PR haha